qiskit/optimization/converters/linear_equality_to_penalty.py

```diff
@@ -0,0 +1,111 @@
+# -*- coding: utf-8 -*-
+
+# This code is part of Qiskit.
+#
+# (C) Copyright IBM 2020.
+#
+# This code is licensed under the Apache License, Version 2.0. You may
+# obtain a copy of this license in the LICENSE.txt file in the root directory
+# of this source tree or at http://www.apache.org/licenses/LICENSE-2.0.
+#
+# Any modifications or derivative works of this code must retain this
+# copyright notice, and modified files need to carry a notice indicating
+# that they have been altered from the originals.
+
+"""Converter to convert a problem with equality constraints to unconstrained with penalty terms."""
+
+import copy
+from typing import Optional
+
+from ..problems import QuadraticProgram, Variable, Constraint, QuadraticObjective
+from ..exceptions.qiskit_optimization_error import QiskitOptimizationError
+
+
+class LinearEqualityToPenalty:
+    """Convert a problem with only equality constraints to unconstrained with penalty terms."""
+
+    def __init__(self):
+        """Initialize the internal data structure."""
```

Comment:
Again comment states the obvious and should be removed as the class one seems to suffice